tools/validators/instance_validator/tests/fake_instances/GOOD/translation_value_range.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Test entity with numeric points.
+
+CHWS_WDT-17-GUID:
+  type: HVAC/CHWS_WDT
+  code: CHWS_WDT-17
+  cloud_device_id: "foobar"
+  translation:
+    return_water_temperature_sensor:
+      present_value: "points.return_water_temperature_sensor.present_value"
+      value_range: 0, 100
+      units:
+        key: "pointset.points.return_water_temperature_sensor.units"
```

Comment:
this does not seem right `pointset.points` but I might be wrong

Reply:
I think this was copied from an existing file but yea I think the correct format should just be "points", but the tests pass regardless